### DISCUSSION TITLE
Favourites Tab

### DISCUSSION THREAD

**Discussion by donacrisante** [neuefische/questions#680] [Web] [answered]
Hello together!
Can someone help me to get my favourites displayed in my EntryList when I click on the favourites tab? I've tried and rewritten my code a lot, but I still can't get the desired result, even though I think I've correctly passed down the props and the corresponding function. Maybe I have overlooked or forgotten something. Thank you in advance!

Here is my PR:
https://github.com/donacrisante/Capstone-project/pull/10

**Comment by donacrisante** [comment]
Command back!! I have just fixed the problem! So I don't need help anymore. Thanks anyway.

**Reply by Kheirah** [reply]
If it's not too much trouble, post your solution here for others to learn from it =)

**Comment by donacrisante** [comment, marked as answer]
Yes, of course. I had forgotten a prop after all. Namely the onClick prop in the Tab component.

<img width="497" alt="Bildschirm­foto 2023-08-18 um 17 10 32" src="https://github.com/neuefische/questions/assets/128093782/b4170618-bc3a-48b2-90f6-3f43a47e2b26">
